functions/go-hello-world/.gitignore

```
@@ -0,0 +1,2 @@
+
```

small nit:
the package in .serverless was commited, I think you can add .serverless here

maybe we could have a top-level gitignore for these kinda of stuff ? I think I started this thing but in retrospect it might be easier to have a top-level .gitignore with all the serverless framework stuff

```suggestion
.serverless/
```
```